static_src/components/users_parent_entity_user_selector.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Action from './action.jsx';
+import FormStore from '../stores/form_store';
+import { Form, FormSelect } from './form';
+import PanelDocumentation from './panel_documentation.jsx';
+import userActions from '../actions/user_actions';
+import { validateString } from '../util/validators';
+
+const USERS_PARENT_ENTITY_USER_FORM_GUID = 'users-parent-entity-users-form';
+
+const propTypes = {
+  userParentEntityUserSelectDisabled: PropTypes.bool,
+  currentUserAccess: PropTypes.bool,
+  parentEntityUsers: PropTypes.array,
+  error: PropTypes.object,
+  parentEntityGuid: PropTypes.string,
+  parentEntity: PropTypes.string,
+  currentEntityGuid: PropTypes.string,
+  currentEntity: PropTypes.string
+};
+const defaultProps = {
+  userParentEntityUserSelectDisabled: false,
+  currentUserAccess: false,
+  error: {}
+};
+
+function stateSetter(props) {
+  return {
+    userParentEntityUserSelectDisabled: props.userParentEntityUserSelectDisabled,
+    currentUserAccess: props.currentUserAccess,
+    parentEntityUsers: props.parentEntityUsers,
+    parentEntityGuid: props.parentEntityGuid,
+    parentEntity: props.parentEntity,
+    currentEntityGuid: props.currentEntityGuid,
+    currentEntity: props.currentEntity,
+    error: props.error
+  };
+}
+
+export default class UsersParentEntityUserSelector extends React.Component {
+  constructor(props) {
+    super(props);
+    FormStore.create(USERS_PARENT_ENTITY_USER_FORM_GUID);
+
+    this.state = stateSetter(props);
+
+    this.validateString = validateString().bind(this);
+    this._onSubmitForm = this._onSubmitForm.bind(this);
+  }
+
+  _onSubmitForm(errs, values) {
+    const entityType = this.state.currentEntity;
+    const entityGuid = this.state.currentEntityGuid;
+    const apiKey = 'auditors';
+    const roles = 'space_auditor';
+    if (values.userGuid) {
+      const userGuid = values.userGuid.value;
+      userActions.addUserRoles(roles, apiKey, userGuid, entityGuid, entityType);
```

Comment:
i think in order to prevent confusion, we should only show org users currently without space roles. then here, upon successful submission, we should remove the user from the list so we don't see them here.